virtualizarr/writers/icechunk.py

```diff
@@ -275,4 +291,5 @@ def write_manifest_virtual_refs(
             location=path.item(),
             offset=offset.item(),
             length=length.item(),
+            checksum=last_updated_at,
```

Comment:
Again an upstream issue, but isn't using a general `checksum` to pass in a time last modified kind of overloading two different things? Can we imagine a case where we wanted to use checksums to verify the integrity of the data but the last-modified was fine, or vice versa?

Reply:
you can do both in icechunk, you can either pass in a datetime, or you can pass in an etag, which is a more effective checksum, but is a lot more work to write into this PR